tests/test_fyle/test_tasks.py

```
@@ -70,3 +70,20 @@
     url = reverse('webhook-callback', kwargs={'workspace_id': 2})
     response = api_client.post(url, data=payload, format='json')
     assert response.status_code == status.HTTP_400_BAD_REQUEST
+
+
+def test_import_expenses(db, create_temp_workspace, add_accounting_export_expenses, add_export_settings, add_fyle_credentials, mocker, api_client, test_connection):
+    """
+    Test import_expenses
+    """
+    expenses_mock = mocker.patch(
```

Ruff (0.8.2)

Local variable `expenses_mock` is assigned to but never used

Remove assignment to unused variable `expenses_mock`

(F841)